src/js/pages/jobs/JobsTable.js

```diff
@@ -105,10 +107,12 @@ class JobsTable extends React.Component {
   }
 
   renderHeadline(prop, job) {
-    const id = encodeURIComponent(job.id);
+    let {id, isGroup, name, schedules} = job;
     let itemImage = null;
+    const jobID = encodeURIComponent(id);
```

Comment:
you could just do `id = encodeURIComponent(id);` so that you don't have to change the lines below.